apps/sms/views/conversation/js/conversation.js

```diff
@@ -3000,7 +3001,7 @@ var ConversationView = {
       type: Compose.type
     });
 
-    Drafts.add(draft);
+    Drafts.add(draft).store();
```

Comment:
Firstly I simply think that we can unify the entry of the draft creation and access into same place, but yeah maybe it's not very clear to increase the complexity for saveDraft.